app/App.js

```diff
@@ -17,9 +17,18 @@ import {
 import RNText from 'app/components/common/RNText'
 import RNButton from 'app/components/common/RNButton'
 import { colors } from 'app/config'
+import config from 'app/config'
+import StorybookUI from 'storybook/'
 
 const App = () => {
   const [count, setCount] = useState(0)
+  if (config.storybooksEnabled) {
+    return (
+      <View style={styles.storybookContainer}>
+        <StorybookUI />
+      </View>
+    )
+  }
```

Comment:
This is ok for the intent in this PR but will need improved as the app builds out/if the app was more complex.